src/A2A/Server/TaskManager.cs

```diff
@@ -183,6 +195,7 @@ public async Task<AgentTask> CreateTaskAsync(string? contextId = null, string? t
             // If the task is found, update its status and history

```

Comment:
As you can see, the existing comment says "update its status and history", but only history was updated so far.

The spec does not mention that sending a message should update the status, but I've assumed that setting it to Working should be fine for Submitted tasks.